bin/git/update-hooks

```diff
@@ -32,14 +73,22 @@ check_update() {
     local dest=$2
     if [ ! -e "$dest" ] ; then
         step_name "Creating new file at '$dest':"
-        git diff --color --no-index /dev/null $src
-        cp $src $dest
+        if [ $verbose == 1 ]; then
```

Comment:
I could, but I think doing so would end up with more lines of code total (because the helper function should bind src and dest locally for documentation purposes), and also could make it harder to determine from the diffs which parts of of the script are changing and which parts are not.

In general, I don't usually feel bad about having two small sections of code be the same.  If it were more than a few lines, or if there were more repetitions then I would definitely be on board for refactoring.  I'm ambivalent about this one - but I'll change it if that's the consensus.